lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -0,0 +1,356 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'moment',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'js/learner_dashboard/models/course_entitlement_model',
+        'js/learner_dashboard/models/course_card_model',
+        'text!../../../templates/learner_dashboard/course_entitlement.underscore',
+        'text!../../../templates/learner_dashboard/verification_popover.underscore'
+    ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             moment,
+             HtmlUtils,
+             EntitlementModel,
+             CourseCardModel,
+             pageTpl,
+             verificationPopoverTpl
+         ) {
+             return Backbone.View.extend({
+                 tpl: HtmlUtils.template(pageTpl),
+                 verificationTpl: HtmlUtils.template(verificationPopoverTpl),
+
+                 events: {
+                     'change .session-select': 'updateEnrollBtn',
+                     'click .enroll-btn': 'handleEnrollChange',
+                     'click .popover-dismiss': 'hideVerificationDialog',
+                     'keydown .final-confirmation-btn': 'handleVerificationPopoverA11y'
+                 },
+
+                 initialize: function(options) {
+                     this.$el = $(this.el);
```

Comment:
In my previous review I noted that this shouldn't be needed. Did removing it break something?

Reply:
Ah, sorry I didn't understand how Backbone binding elements worked - you are right, removed it.